### ISSUE TITLE
Electrical status epilepticus during slow-wave sleep

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7142]
**Preferred term label:**
Electrical status epilepticus during slow-wave sleep

@DLewisSmith   -- is this a valid term? Where would it go?

**Synonyms**

**Definition (free text, please give PubMed ID)**
The occurrence of continuous spike and slow waves during non-rapid-eye-movement (non-REM) sleep, the paroxysmal abnormalities being substantially less frequent during the awake state and REM sleep.
PMID:1918335

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Epilepsy, focal, with speech disorder and with or without impaired intellectual development
[OMIM:245570](https://omim.org/entry/245570)

PMID: 23933820
19 of 33 individuals for which information was available (see table with Supplementary Note. Clinical data.)

**Your nano-attribution (ORCID)**

**Comment by DLewisSmith** [comment]
This is a synonym of https://hpo.jax.org/app/browse/term/HP:0031491

**Comment by pnrobinson** [comment]
thanks, I added the synonym to the term!
